src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -339,11 +339,19 @@ ov::AnyMap get_baseline_common_config() {
         {"NPUW_WEIGHTS_BANK", "shared"},
         {"NPUW_SLICE_OUT", "YES"},
         {"NPUW_FUNCALL_ASYNC", "YES"}};
+    // FIXME: this config logic is getting more and more complex
+    if (npudesc.has_value() && npudesc->compiler_dq) {
+        config.emplace("NPUW_DQ", "YES");
+        config.emplace("NPUW_DQ_FULL", "NO");
+        config.emplace("NPU_COMPILER_DYNAMIC_QUANTIZATION", "YES");
+        config.erase("NPUW_DCOFF_TYPE");
```

Comment:
I thought it's not needed when `NPUW_DQ: YES`.

Reply:
Needed in case config.emplace("NPUW_DQ_FULL", "NO");